dipy/workflows/io.py

```diff
@@ -97,3 +98,95 @@ def run(self, input_files,
                              .format(ncl1))
 
         np.set_printoptions()
+
+
+class FetchFlow(Workflow):
+
+    @classmethod
+    def get_short_name(cls):
+        return 'fetch'
+
+    def load_module(self, module_path):
+        """Load / reload an external module.
+
+        Parameters
+        ----------
+        module_path: string
+            the path to the module relative to the main script
+
+        Returns
+        -------
+        module: module object
+
+        """
+        if module_path in sys.modules:
+            return importlib.reload(sys.modules[module_path])
+        else:
+            return importlib.import_module(module_path)
+
+    def run(self, data_names, out_dir=''):
+        """Download files to folder and checks their md5 checksums.
```

Comment:
Either "Downloads files to folder and checks their md5 checksums" or "Download files to folder and check their md5 checksums"